### ISSUE TITLE
Expo 49 - Breaking Changes

### ISSUE THREAD

**Issue by hazirmagron** [enesozturk/react-native-hold-menu#111]
**Describe the bug**
The overlay is completely broken since I updated to Expo@49. 

The 'HoldItem' child components still animate when pressed. The overlay and menu never show up, however, and the child component that was pressed is frozen on the screen.

**Package versions**
 - React: "18.2.0"
 - React Native: "0.72.3"
 - React Native Reanimated: "~3.3.0"
 - React Native Hold Menu: "^0.1.6"
 - Expo:  "^49.0.0"

**Comment by holma91** [comment]
Is this fixed yet?

**Comment by pavelbabenko** [comment]
Same issue

**Comment by raymond-tetteh** [comment]
I have the same issue.

**Comment by mimo-10** [comment]
got the same issue her, any solutions suggested?

**Comment by Nasseratic** [comment]
Hello guys 👋🏼  

I am working on a fix here 🤓  https://github.com/enesozturk/react-native-hold-menu/pull/112

It's WIP, but I got everything working on my project

**Comment by moedeveloper** [comment]
Unfortunately same issue :(

**Comment by sssst9s** [comment]
Hey, is there a fixed version ? I saw the new PR being WIP and changes seems to works perfecly. Did you already publish it ? Or still WIP ? Thanks ✌️

**Comment by Nasseratic** [comment]
Hey @devmalcolm 

Yeah, it should be ready. But I can't publish it as I am not a contributor. Will try to bring someone to approve my PR. For now feel free to install the version in the PR by:
```
npm i https://github.com/Nasseratic/react-native-hold-menu-upgrade-expo-49.git#expo-49-needed-fixes
```

**Comment by viktor32** [comment]
@Nasseratic great job, thank you!

But I think the correct command to install from your repo is (with the branch where the fix)
```
npm i https://github.com/Nasseratic/react-native-hold-menu-upgrade-expo-49.git#expo-49-needed-fixes
```

**Comment by BennoCrafter** [comment]
Hey @Nasseratic! I've downloaded your updated version, but sadly it still doesnt work. Is it just me, or is it because there is a new version?

**Comment by MathiasSvDK** [comment]
Same issue. Using expo 50

**Comment by mimo-10** [comment]
It works, you only need to use this version 
`npm i https://github.com/Nasseratic/react-native-hold-menu-upgrade-expo-49.git#expo-49-needed-fixes`
And you need to make a change inside the backdrop component in order to make it work on android.
Just change View, to AnimatedBlurView

**Comment by BennoCrafter** [comment]
Just for android?

**Comment by mimo-10** [comment]
Yes, you need to do that only for android, for ios it works without any changes.

**Comment by tconroy** [comment]
Im having trouble trying to install this fork.. 🤔 any ideas?

```
❯ npm i https://github.com/Nasseratic/react-native-hold-menu-upgrade-expo-49.git#expo-49-needed-fixes
zsh: no matches found: https://github.com/Nasseratic/react-native-hold-menu-upgrade-expo-49.git#expo-49-needed-fixes
```

**Comment by Nasseratic** [comment]
@tconroy It works for me 😅

**Comment by tconroy** [comment]
> @tconroy It works for me 😅 

Yes I just needed to wrap the url as a string and now it works 😊

**Comment by daxaxelrod** [comment]
Can we merge the upgrade to 49 (also confirming nasseratic's fix works on expo 50) to main

**Comment by BraeschAntoine** [comment]
Thx your solution worked perfectly, your fork saved my day ! I don't understand why this isn't already merged
